### ISSUE TITLE
Upgrade issues from v3

### ISSUE THREAD

**Issue by Ylmzef** [moxi9/phpfox#131]
In signup forum there html codes like following:

Fisrtname type="text" name="val[first_name]" id="first_name" value="" size="30" />

**Comment by Data66PF** [comment]
Make sure you are not using a third party theme. If you are, contact that developer.

**Comment by Ylmzef** [comment]
Its default v4 theme.

**Comment by Data66PF** [comment]
Did you make any changes or load any v3 stuff?

**Comment by Ylmzef** [comment]
No, just upgraded fom V3.8.0

**Comment by Data66PF** [comment]
Then you have some V3 addons and themes on the site? Try disabling v3 addons and plugins and see if that works. I'm not seeing issues on my upgrades.

**Comment by Ylmzef** [comment]
I cant see v3 modules and themes in v4

**Comment by Data66PF** [comment]
Click Maintenance when in Admincp.

**Comment by Ylmzef** [comment]
Nothing there under Toggle Modules only core modules.

**Comment by Ylmzef** [comment]
Also cant remove "Anti Spam Questions"

**Comment by Ylmzef** [comment]
Also footer links removed after upgrade, now only showing contact ys and advertise.

**Comment by Ylmzef** [comment]
I found following errors in site error logs:

http://pastebin.com/RFQzcSTa

**Comment by Data66PF** [comment]
Is that for v4.0.3 released today? Please also note:

FYI, for some having install, upgrade issues of the script, apps or themes , check out https://dev.mysql.com/doc/refman/5.0/en/sql-mode.html
By default SQL mode is not set to strict mode so for those that have it set to that mode have run into problems.

**Comment by Ylmzef** [comment]
I have it still on v4.0.3

**Comment by Data66PF** [comment]
As mentioned, please check your server to see if it is in SQL strict mode. Then read the link I provided.

**Comment by Ylmzef** [comment]
Also i have enabled it.

**Comment by Data66PF** [comment]
Don't enable strict mode. That is what is causing issues.

**Comment by Ylmzef** [comment]
Its now disabled but still the issue: http://pastebin.com/5WiBnS2b

**Comment by Ylmzef** [comment]
Also cant remove any apps.

**Comment by Data66PF** [comment]
Your ticket #109  says it's fixed so I will close this for the upgrade issue. Apps are a different issue.

**Comment by Ylmzef** [comment]
It was about installation.
This for bugs after upgrade. Please reopen this ticket.

**Comment by Data66PF** [comment]
We don't want multiple issues in one ticket. Please only have one issue. This one is too hard to follow now what the issue is. Please open a ticket with only one issue in it. You can open more than one ticket if you have more than one issue but please make sure before you upgrade that you disabled all third party products.

**Comment by Ylmzef** [comment]
It is not multiple issues, its only one issue and that is upgrade issue from script.

**Comment by Data66PF** [comment]
You had several issues listed from "can't install apps" to code in registration, to "bugs after upgrade" to  other errors showing. Please wait for v4.0.4 being released maybe tomorrow.

**Comment by Ylmzef** [comment]
All the mentioned bugs due to the upgrade script.
Also i have not disabled third party products it was not mentioned in upgrade guide page.
How can i now disable it?
